enterprise/internal/database/perms_store_test.go

```diff
@@ -2378,6 +2379,16 @@ INSERT INTO user_external_accounts(user_id, service_type, service_id, account_id
 		} else if userIDs["bob_gitlab"] != 2 {
 			t.Fatalf(`userIDs["bob_gitlab"]: want 2 but got %d`, userIDs["bob_gitlab"])
 		}
+
+		accounts = &extsvc.Accounts{
+			ServiceType: "github",
+			ServiceID:   "https://github.com/",
+			AccountIDs:  []string{"cindy_github"},
+		}
+		userIDs, err = s.GetUserIDsByExternalAccounts(ctx, accounts)
+
+		require.Nil(t, err)
+		require.Empty(t, userIDs)
```

Comment:
nit
```suggestion
		require.Nil(t, err)
		assert.Empty(t, userIDs)
```

My personal principle of using `require` is when test just can't continue.